hybrid/layout/pv_layout.py

```diff
@@ -213,8 +213,8 @@ def set_system_capacity(self,
         """
         Changes system capacity in the existing layout
         """
-        if type(self.parameters) == PVGridParameters:
-            self.reset_solargrid(size_kw, self.parameters)
+        if type(self.parameters) == PVGridParameters or "DetailedPVParameters" in str(type(self.parameters)):
```

Comment:
If someone named their parameters differently, then this code wouldn't be flexible enough to handle it. I think we should throw an error to stave off confusion

Reply:
I'm not sure now why I even needed to accommodate the `DetailedPVParameters` type, so I just removed it. I think the tests will pass, we'll see.